frontend/app_flowy/lib/plugins/board/presentation/toolbar/board_toolbar.dart

```diff
@@ -47,14 +48,22 @@ class _SettingButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final theme = context.read<AppTheme>();
-    return FlowyIconButton(
-      hoverColor: theme.hover,
-      width: 22,
-      onPressed: () => BoardSettingList.show(context, settingContext),
-      icon: Padding(
-        padding: const EdgeInsets.symmetric(vertical: 3.0, horizontal: 3.0),
-        child: svgWidget("grid/setting/setting"),
+    return Popover(
+      triggerActions: PopoverTriggerActionFlags.click,
+      child: FlowyIconButton(
+        hoverColor: theme.hover,
+        width: 22,
+        onPressed: () {},
+        icon: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 3.0, horizontal: 3.0),
+          child: svgWidget("grid/setting/setting"),
+        ),
       ),
+      popupBuilder: (BuildContext popoverContext) {
```

Comment:
The popupBuilder can provide a built-in OverlayContainer, develop can decide whether to use it.
 If so, there is no need to wrap every popup child with OverlayContainer.

Reply:
The popover widget provides the functionalities that the popover needs.
It's better not to define the style of the popover. 
`OverlayContainer` embeds the Theme and the style of the popover.
Especially the `AppTheme`, this class can't be added to the third-party package.
If the user doesn't want to warp every into a container, they can warp the popover into another popover such as a `AppFlowyStylePopover` in the `flowy_infra_ui` package.

Reply:
> AppFlowyStylePopover

`AppFlowyStylePopover` is exactly what I mean, we should use AppFlowyStylePopover which is just a wrapper of `Popover`, but it implements the appflowy style. The caller does not need to wrap the OverlayContainer anymore.